AUTHORS.md

```diff
@@ -0,0 +1,231 @@
+
+[@zhangcoldwater](https://github.com/zhangcoldwater)
+
+[@fuxiaohei](https://github.com/fuxiaohei)
+
+[@qhsong](https://github.com/qhsong)
+
+[@roger-lijian](https://github.com/roger-lijian)
+
+[@yangwenmai](https://github.com/yangwenmai)
+
+[@arthuryangcs](https://github.com/arthuryangcs)
+
+[@liubin](https://github.com/liubin)
+
+[@hnakamur](https://github.com/hnakamur)
+
+[@hzmangel](https://github.com/hzmangel)
+
+[@hsqlu](https://github.com/hsqlu)
+
+[@Alienero](https://github.com/Alienero)
+
+[@alston111111](https://github.com/alston111111)
+
+[@cuiyuan](https://github.com/cuiyuan)
+
+[@fudali113](https://github.com/fudali113)
+
+[@knarfeh](https://github.com/knarfeh)
+
+[@lkk2003rty](https://github.com/lkk2003rty)
+
+[@GregoryIan](https://github.com/GregoryIan)
+
+[@silenceper](https://github.com/silenceper)
+
+[@woodpenker](https://github.com/woodpenker)
+
+[@BusyJay](https://github.com/BusyJay)
+
+[@foobar](https://github.com/foobar)
+
+[@8cbx](https://github.com/8cbx)
+
+[@AkihiroSuda](https://github.com/AkihiroSuda)
+
+[@hamo](https://github.com/hamo)
+
+[@kiroInn](https://github.com/kiroInn)
+
+[@chuchao333](https://github.com/chuchao333)
+
+[@colelawrence](https://github.com/colelawrence)
+
+[@xiaods](https://github.com/xiaods)
+
+[@tangdi](https://github.com/tangdi)
+
+[@gelstudios](https://github.com/gelstudios)
+
+[@mrmiywj](https://github.com/mrmiywj)
+
+[@shuhuai007](https://github.com/shuhuai007)
+
+[@kwo](https://github.com/kwo)
+
+[@EndlessIdea](https://github.com/EndlessIdea)
+
+[@liyu4](https://github.com/liyu4)
+
+[@Librazy](https://github.com/Librazy)
+
+[@myrice](https://github.com/myrice)
+
+[@ts25504](https://github.com/ts25504)
+
+[@simon-xia](https://github.com/simon-xia)
+
+[@mtunique](https://github.com/mtunique)
+
+[@bom-d-van](https://github.com/bom-d-van)
+
+[@WangXiangUSTC](https://github.com/WangXiangUSTC)
+
+[@yangxuanjia](https://github.com/yangxuanjia)
+
+[@Xuanwo](https://github.com/Xuanwo)
+
+[@losas](https://github.com/losas)
+
+[@codeworm96](https://github.com/codeworm96)
+
+[@Zhoubirdblue](https://github.com/Zhoubirdblue)
+
+[@dawxy](https://github.com/dawxy)
+
+[@framlog](https://github.com/framlog)
+
+[@winkyao](https://github.com/winkyao)
+
+[@hiwjd](https://github.com/hiwjd)
+
+[@liugangnhm](https://github.com/liugangnhm)
+
+[@JinheLin](https://github.com/JinheLin)
+
+[@lijin](https://github.com/lijin)
+
+[@luckcolors](https://github.com/luckcolors)
+
+[@maxwell92](https://github.com/maxwell92)
+
+[@queenypingcap](https://github.com/queenypingcap)
+
+[@tennix](https://github.com/tennix)
+
+[@ueizhou](https://github.com/ueizhou)
+
+[@v01dstar](https://github.com/v01dstar)
+
+[@weekface](https://github.com/weekface)
+
+[@zs634134578](https://github.com/zs634134578)
+
+[@qiukeren](https://github.com/qiukeren)
+
+[@hawkingrei](https://github.com/hawkingrei)
+
+###### Auto generated by [gaocegege/maintainer](https://github.com/gaocegege/maintainer) on 2017-05-22
```

Comment:
This is auto-generated, if remove, remember to remove the text every time we update the doc.